rust/datafusion/src/physical_plan/hash_join.rs

```diff
@@ -447,6 +447,11 @@ pub(crate) fn create_key(
                 // store the string value
                 vec.extend_from_slice(value.as_bytes());
             }
+            DataType::Boolean => {
+                let array = col.as_any().downcast_ref::<BooleanArray>().unwrap();
+                let x: u8 = if array.value(row) { 1 } else { 0 };
```

Comment:
This could probably also use `array.value(row) as u8`?

Reply:
Thanks